app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -1,5 +1,9 @@
 <% title t('titles.enter_2fa_code.one_time_code') %>
 
+<%= render AlertComponent.new(type: :info, class: 'margin-bottom-4 usa-alert--info-time') do %>
+  <%= render CountdownComponent.new(expiration: @presenter.otp_expiration) %> remaining
```

Comment:
👀 untranslated `remaining`?
```suggestion
  <%= render CountdownComponent.new(expiration: @presenter.otp_expiration) %>
```

Reply:
Yep, I need to get translations for this.